### ISSUE TITLE
AMQP 1.0 Java Client for RabbitMQ

### ISSUE THREAD

**Issue by hrirm** [rabbitmq/rabbitmq-java-client#652]
Hello,

is there a plan to provide an enhancement in the current client to use protocol AMQP 1.0  rather than using QPid Client which didn't work for me and SwiftMQ which not totally compatible

* RabbitMQ version : **3.8.4**
* Erlang version : **23**

Thanks in advance for your support

**Comment by lukebakken** [comment]
No, this is not planned as far as I know.

> using QPid Client which didn't work for me and SwiftMQ which not totally compatible

It would be helpful if you provided more information about exactly what you tried, and what didn't work. Please follow-up via the [`rabbitmq-users`](https://groups.google.com/forum/#!forum/rabbitmq-users) mailing list, not GitHub.

**Comment by michaelklishin** [comment]
Duplicate of #651.

**Comment by hrirm** [comment]
Thanks for your support. It's very sad to start thinking to migrate to another broker

**Comment by michaelklishin** [comment]
We'd appreciate some details on how existing AMQP 1.0 clients are "not compatible" on the mailing list. You have 4 protocols to choose from when using RabbitMQ, including this client and AMQP 0-9-1. They are not going away after 13 years of production usage.

**Comment by hrirm** [comment]
Hello my issue is with the connection to a vhost:

in the QPid JMS Client I set the vhost name in the URI like the following : 
amqp://localhost:5672?jms.clientID=foo&amqp.vhost=vhost_externe:localhost

the user is already created in RabbitMQ.

2020-06-10 10:08:03.543 [error] <0.1278.0> AMQP 1.0: user 'user.externe' is not allowed to access virtual host '/'
2020-06-10 10:08:03.544 [error] <0.1286.0> CRASH REPORT Process <0.1286.0> with 0 neighbours exited with reason: not_allowed in gen_server2:init_it/6 line 600
2020-06-10 10:08:03.544 [error] <0.1278.0> AMQP 1.0 connection <0.1278.0> (running), channel 0 - error:
<<"Access for user 'user.externe' was refused: insufficient permissions\n">>

It doesn't recognize the vhost that I mentionned. I have followed the instruction in the following link:

https://github.com/rabbitmq/rabbitmq-amqp1.0#virtual-hosts 

did I missed something to configure in the java client?

PS:
if I set permission to the user to the vhost "/" the connextion is established but this not what I need
